tests-integration/src/database.rs

```diff
@@ -127,12 +128,10 @@ impl Database {
     }
 
     pub async fn insert(&self, requests: InsertRequests) -> Result<u32> {
-        let _timer = metrics::METRIC_GRPC_INSERT.start_timer();
```

Comment:
As `Database` only tend to be used in `test` parts, so I remove `metrics` parts (it is a private module, is it good not to expose it?)

Reply:
That's ok, metrics are not required here.